docs/site/Authentication-overview.md

```diff
@@ -0,0 +1,68 @@
+---
+lang: en
+title: 'LoopBack 4 Authentication Overview'
+keywords: LoopBack 4.0, LoopBack 4, Authentication
+sidebar: lb4_sidebar
+permalink: /doc/en/lb4/Authentication-overview.html
+---
+
+Security is of paramount importance when developing a web or mobile application
+and usually consists of two distinct pieces:
+
+- Authentication
+- Authorization
+
+Authentication is a process of verifying someone's identity before a protected
+resource is accessed.
```

Comment:
Authentication is the process of validating the user/entity to the system, which enables authorised access to the protected routes.